src/transformers/generation/logits_process.py

```diff
@@ -1064,8 +1064,8 @@ class SequenceBiasLogitsProcessor(LogitsProcessor):
     </Tip>
 
     Args:
-        sequence_bias (`Dict[Tuple[int], float]`):
-            Dictionary that maps a sequence of tokens to its bias term. Positive biases increase the odds of the
+        sequence_bias (`List[List[Union[List[int], float]]]`):
+            List of lists that maps a sequence of tokens to its bias term. Positive biases increase the odds of the
```

Comment:
```suggestion
            List of lists that maps a sequence of tokens to its bias term (e.g. `[[[10, 45], -2.0], [[64], -7.5]]`). Positive biases increase the odds of the
```

Since the format is more complex, let's add a concrete example right at the start :) That way the format should be immediately obvious

Reply:
sure, done